### ISSUE TITLE
[MU4 Issue] Cross staff beaming is screwed up

### ISSUE THREAD

**Issue by worldwideweary** [musescore/MuseScore#9960]
**Describe the bug**
Cross-staff beams need some work.
Not sure if this is just partially implemented and left hangin' or was screwed up in a previous attempt to be thorough

**To Reproduce**
Use cross-staff shortcut upon 8th notes from bass to treble clef:

![MS-crossstaffbeam](https://user-images.githubusercontent.com/7139517/144519056-c13b8c9e-67e9-498e-898b-f1df0bd2d6cf.png)

Obviously should be like MS3
![MS3-crossstafbeam](https://user-images.githubusercontent.com/7139517/144519321-9052bafb-3918-44f7-ada5-bdf5ff736e97.png)
:

**Comment by oktophonie** [comment]
This is a consequence of #9792 (as the description there mentions, cross-staff beaming remains to be implemented). This will be fixed shortly as part of this ongoing overhaul.

**Comment by worldwideweary** [comment]
Ok cool. Unless suggested otherwise, I'll keep this open as a separate issue for now instead of closing it so that after #9792 is finalized, the cross-staff beaming can be tested in and of itself (or if there's anything else funny going on with it afterwards).
